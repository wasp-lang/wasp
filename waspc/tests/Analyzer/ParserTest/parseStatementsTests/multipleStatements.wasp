constant Pi 3.14159
constant E  2.71828
