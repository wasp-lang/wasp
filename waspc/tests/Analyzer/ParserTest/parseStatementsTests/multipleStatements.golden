(AST
  (Decl@1:1-19 type=constant name=Pi
    (Double@1:13-19 value=3.14159)
  )
  (Decl@2:1-19 type=constant name=E
    (Double@2:13-19 value=2.71828)
  )
)
